.github/scripts/check_advanced_requirement.sh

```
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# 1. Check for Exemptions (Triage, Committers, Maintainers)
+# Using 'triage', 'write', or 'admin' covers all core teams.
+PERM_JSON=$(gh api repos/$REPO/collaborators/$ASSIGNEE/permission)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)